### ISSUE TITLE
Math keys (`+`, `-`, `*`, `/`) on numpad does not work

### ISSUE THREAD

**Issue by tobiasBora** [akermu/emacs-libvterm#437]
Hello,

I don't know why, but when I press the keys `+`, `-`, `*`, `/` on the numpad, the press is not taken into account (but the dot and the numbers do work). This is very disturbing, especially because I like to use the `-` from the numpad to type arguments like `ls -al`. Emacs does recognize these keys without any issue. If it helps, here are the `xev` outputs when I press this key:

```
KeyPress event, serial 40, synthetic NO, window 0xb600001,
    root 0x1bf, subw 0x0, time 80414722, (70,112), root:(70,978),
    state 0x10, keycode 82 (keysym 0xffad, KP_Subtract), same_screen YES,
    XLookupString gives 1 bytes: (2d) "-"
    XmbLookupString gives 1 bytes: (2d) "-"
    XFilterEvent returns: False

KeyRelease event, serial 40, synthetic NO, window 0xb600001,
    root 0x1bf, subw 0x0, time 80414871, (70,112), root:(70,978),
    state 0x10, keycode 82 (keysym 0xffad, KP_Subtract), same_screen YES,
    XLookupString gives 1 bytes: (2d) "-"
    XFilterEvent returns: False
```

Thanks in advance!

**Comment by Sbozzolo** [comment]
I haven't had a numpad in a long time, so I cannot test this directly, but looking here:
[at Xah Lee's page](http://ergoemacs.org/emacs/emacs_bind_number_pad_keys.html) see that Emacs treats numpad keys differently, so we should probably bind the keys to make it work.

Can you try something like:
```emacs-lisp
(define-key vterm-mode-map (kbd "<kp-add>")   #'vterm--self-insert)
```

**Comment by tobiasBora** [comment]
Thanks a lot for your answer. So your solution does not work directly (nothing happens when I press the `+`), but I was able to make it work using `vterm-send-key` instead:

```
(define-key vterm-mode-map (kbd "<kp-add>")
    (lambda () (interactive) (vterm-send-key "+")))
```

I was able to use similar changes to make `kp-subtract`, `kp-multiply` and `kp-divide` work. I don't know if it's supposed to work in all situations, but it does the job for now!
